core-schema/src/main/resources/sql-schema/V5__Mod_credits.sql

```diff
@@ -0,0 +1,11 @@
+
+CREATE TABLE credits (
+  user_id INT PRIMARY KEY,
+  balance NUMERIC(12, 3) NOT NULL,
+  FOREIGN KEY (user_id) REFERENCES user_ids (id) ON DELETE CASCADE
+);
+
+CREATE VIEW credits_with_names AS
```

Comment:
what is the point of this?

Reply:
It's used to display baltop info. I have found that selecting from views which join is somewhat more manageable than selecting joins. The database optimizes it to the same query plan anyway.